### ISSUE TITLE
some doubts about the validation set

### ISSUE THREAD

**Issue by carbonatedbeverages** [llmunlearningsemeval2025/SEMEval2025-LLM-Unlearning#2]
Hello organizer, I have some doubts about the validation set. I noticed that the data in the validation set has not appeared in the training set. How can I use the validation set to prove whether my model has forgotten the content in my training set? Or do I need to divide a part of the test set from the training set myself? Am I misunderstanding something?

**Comment by anilkram** [comment]
Hello,
The fine tuned models have been trained to memorize both the train and validation set (you're welcome to test this on the models by prompting them with questions in validation set - you will see perfect regurgitation). For the challenge please use the validation set to tune any hyperparameters in your algorithm (as needed).
